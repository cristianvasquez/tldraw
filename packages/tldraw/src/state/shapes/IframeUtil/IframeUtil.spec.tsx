import { Iframe, Image } from '..'

describe('Iframe shape', () => {
  it('Creates a shape', () => {
    expect(Iframe.create({ id: 'iframe' })).toMatchSnapshot('iframe')
  })
})
