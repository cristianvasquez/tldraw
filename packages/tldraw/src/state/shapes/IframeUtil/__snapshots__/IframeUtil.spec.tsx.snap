// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iframe shape Creates a shape: iframe 1`] = `
Object {
  "assetId": "assetId",
  "childIndex": 1,
  "id": "iframe",
  "name": "Iframe",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": true,
    "scale": 1,
    "size": "small",
  },
  "type": "iframe",
}
`;
